gradle-plugin/plugin/src/main/java/com/yelp/codegen/plugin/FeatureConfiguration.kt

```diff
@@ -1,11 +1,11 @@
 package com.yelp.codegen.plugin
 
+import org.gradle.api.Project
+import org.gradle.api.model.ObjectFactory
+import org.gradle.api.provider.ListProperty
 import org.gradle.api.tasks.Input
 import org.gradle.api.tasks.Optional
 
-open class FeatureConfiguration {
-
-    @Input
-    @Optional
```

Comment:
I'm not sure why this was needed. I don't think something else than task properties can be annotated.

Reply:
It turns out this was into a @Nested task property so annotation are needed. IntelliJ complained because it required `@gete:Input` and not just `Input`